app/models/airline.rb

```diff
@@ -7,4 +7,17 @@ class Airline < ActiveRecord::Base
   def self.search(query)
     where("name ilike :q or description ilike :q", q: "%#{query}%")
   end
+
+  def ratings
+    ratings_array = []
+    reviews.each do |review|
+      ratings_array << review.rating
+    end
+    ratings_array
+  end
+
+  def average_rating
+    sum = (ratings.inject (0) { |total, n| total + n }).to_f
+    sum / ratings.count
```

Comment:
What happens if ratings.count is 0? I think you are leaving yourself open to a potential bug here if this method could ever get called when there are no ratings. I'd add unit test coverage for this edge case.